src/commands/general/yourcontrols.ts

```diff
@@ -0,0 +1,12 @@
+import { CommandDefinition } from '../../lib/command';
+import { CommandCategory } from '../../constants';
+
+export const yourControls: CommandDefinition = {
+    name: ['yourcontrols', 'yc'],
+    description: 'Provides link to the YourControls Discord server',
+    category: CommandCategory.GENERAL,
+    executor: async (msg) => {
+        await msg.channel.send('https://discord.gg/p7Bzcv3Yjd');
+        return msg.delete();
```

Comment:
Why delete the user message?

Reply:
idk ben told me to

Reply:
@benw202

Reply:
Was done on another invite link, MSFS, feel free to remove on both :)

Reply:
No need to delete user message, please remove async function also as it is unnecessary. :)